src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,123 @@
+import { FC, useCallback, useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Search } from "@mui/icons-material";
+import classNames from "classnames";
+
+import defaultGameThumbnailImage from "@assets/images/defaultGameThumbnail.png";
+
+import "./SearchBar.css";
+
+import { useFetch } from "@/api/privateApi";
+import ImageWithLoader from "@/components/ui/Image/ImageWithLoader";
+import { Loader } from "@/components/ui/Loader/Loader";
+import { TLabel, TText } from "@/components/ui/TranslationLabel/TLabel";
+import { useOutsideClick } from "@/hooks/useOutsideClick";
+import { GameType } from "@/types/Game";
```

Comment:
```suggestion
import { useFetch } from "@api/privateApi";
import ImageWithLoader from "@components/ui/Image/ImageWithLoader";
import { Loader } from "@components/ui/Loader/Loader";
import { TLabel, TText } from "@components/ui/TranslationLabel/TLabel";
import { useOutsideClick } from "@hooks/useOutsideClick";
import { GameType } from "@customTypes/Game";
```
**Suggestion** : Remonte plus haut les imports